### PR TITLE
Create michael-town.yaml

### PR DESCRIPTION
**Pull request by michaeltown-phd**

### CODE REVIEW

File: team/michael-town.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Michael (Mike) Town (he/him)
+  avatar: https://avatars.githubusercontent.com/u/80553650?v=4
+  role: Research Scientist
+  organizations:
+    - name: Earth and Space Research
+  bio:
+ I am an innovative research scientist, educator, and instructional coach with professional experience in machine learning and data analysis, computational modeling, remote field work, and secondary and university-level curriculum development. Equity and social justice are at the center of both my research and education efforts.
```
Comment (JessicaS11) [inline team/michael-town.yaml]: @michaeltown-phd It looks like the indentation is off here (and is causing the build to fail).

Comment (michaeltown-phd) [reply team/michael-town.yaml]: OK, I think I fixed it. I ran a 'preview' and that seemed to succeed. Sorry!

File: team/michael-town.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Michael (Mike) Town (he/him)
+  avatar: https://avatars.githubusercontent.com/u/80553650?v=4
+  role: Research Scientist
+  organizations:
+    - name: Earth and Space Research
+  bio:
+    - I am an innovative research scientist, educator, and instructional coach with professional experience in machine learning and data analysis, computational modeling, remote field work, and secondary and university-level curriculum development. Equity and social justice are at the center of both my research and education efforts.
```
Comment (scottyhq) [inline team/michael-town.yaml]:
```suggestion
  bio: >
    I am an innovative research scientist, educator, and instructional coach with professional experience in machine learning and data analysis, computational modeling, remote field work, and secondary and university-level curriculum development. Equity and social justice are at the center of both my research and education efforts.
```

**Review by scottyhq — CHANGES_REQUESTED** [review]
See above suggestion to get the multiline bio to render (https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210)

**Comment by ZihengSun** [comment]
The issue is that the name cannot have any special characters like brackets. Please remove the brackets and the popup intro will work again.

File: team/michael-town.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Michael (Mike) Town (he/him)
```
Comment (ZihengSun) [inline team/michael-town.yaml]:
```suggestion
- title: Michael Mike Town
```

**Comment by JessicaS11** [comment]
> The issue is that the name cannot have any special characters like brackets. Please remove the brackets and the popup intro will work again.

@ZihengSun - @jomey fixed this for us in #57!

@michaeltown-phd There's now also a `pronouns` option! Thanks for suggesting we add that.

File: team/michael-town.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  avatar: https://avatars.githubusercontent.com/u/80553650?v=4
+  role: Research Scientist
+  organizations:
+    - name: Earth and Space Research
+  bio: >
```
Comment (JessicaS11) [inline team/michael-town.yaml]:
```suggestion
    - name: Earth and Space Research
  pronouns: he/him
  bio: >
```

File: team/michael-town.yaml (hunk 3 above)
Comment (JessicaS11) [inline team/michael-town.yaml]:
```suggestion
- title: Michael (Mike) Town
```

**Review by michaeltown-phd — COMMENTED** [review]
this looks good to me. Thanks for taking care of it!

**Comment by michaeltown-phd** [comment]
Is there anything more for me to do here? Or can you publish it?

**Comment by scottyhq** [comment]
the 'read more' bio link still isn't rendering so we'll have to adjust something, but nothing more to do on your part @michaeltown-phd  :)

File: scripts/build_team_yaml.sh [hunk 5]
```diff
@@ -14,7 +14,8 @@ find . -type f -name '*.yaml' \
   sort | xargs -I '{}' cat '{}' > team_people.yaml
 
 # Indent to proper yaml
-sed -i 's/^/  /' team_people.yaml
+sed -i.bkp -e 's/^/  /' team_people.yaml
```
Comment (scottyhq) [inline scripts/build_team_yaml.sh:17]: (from https://github.com/uwhackweek/jupyterbook-template/pull/167)

**Review by scottyhq — APPROVED** [review]
just needed to merge with the main branch which has @jomey 's great addition of the pronouns field. Thanks all!

**Pull request merged**